include/common_structs.h

```diff
@@ -805,12 +805,17 @@ typedef struct TextureHeader {
     /* 0x2F */ u8 filtering;
 } TextureHeader; // size = 0x30
 
+enum BattleSubmenu {
+    BATTLE_SUBMENU_HAMMER       = 1,
+    BATTLE_SUBMENU_JUMP         = 2,
+};
+
```

Comment:
Should probably add put this into `enums.h` instead and also add another one for the submenuStatuses